src/blockrelay/graphene_set.cpp

```diff
@@ -125,7 +125,7 @@ double CGrapheneSet::OptimalSymDiff(uint64_t nBlockTxs, uint64_t nReceiverPoolTx
     auto fpr = [nReceiverPoolTx, nBlockAndReceiverPoolTx](uint64_t a) {
         float _fpr = a / float(nReceiverPoolTx - nBlockAndReceiverPoolTx);
 
-        return _fpr < 1.0 ? _fpr : FILTER_FPR_MAX;
+        return _fpr < FILTER_FPR_MAX ? _fpr : FILTER_FPR_MAX;
     };
```

Comment:
Comparing to 1.0 was an oversight on my part. I don't see any problem with making this change. Obviously, FILTER_FPR_MAX is quite close to 1.0 anyhow. But also, forcing higher FPRs like 0.9999 to be equal to FILTER_FPR_MAX in this method should only affect the (size) optimality of the choice of IBLT size. The actual FPR for the Bloom filter is set further downstream. So I don't regard it as a particularly high-risk change.